source/ota_over_mqtt_demo.c

```diff
@@ -80,6 +76,10 @@
     #error "Please define the democonfigCLIENT_IDENTIFIER with the thing name for which OTA is performed"
 #endif
 
+#if !defined( MQTT_AGENT_NETWORK_BUFFER_SIZE ) || ( MQTT_AGENT_NETWORK_BUFFER_SIZE < OTA_DATA_BLOCK_SIZE )
+    #error "Please define MQTT_AGENT_NETWORK_BUFFER_SIZE to adequate size as required by OTA data block"
```

Comment:
Could we reword this to be a little clearer?  e.g., buffer is too small, please increase the size to at least x...

Reply:
Reworded to be more clear.